<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>UIExtractor</author>
 <comment>This is a comment</comment>
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>831</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>&amp;Search for:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>searchEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="toolTip">
        <string>Search pattern. Use wildcards “*” and “?” as desired.</string>
       </property>
       <property name="whatsThis">
        <string>Examine various fields to see if they contain the search pattern: The pattern “meet” would match “kick-off meeting”. Wildcards “*” (matches any number of characters) and “?” (matches any single character) are supported.  For example, the search pattern “meet*boss” matches “Meeting with boss”. The pattern “c?t” would match “cat”, “cot”, and so on.</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="searchTypeGroup">
     <property name="title">
      <string>Search For</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="eventsCheck">
        <property name="toolTip">
         <string>Search for events with matching criteria.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to search for events that match the search pattern and have start dates or end dates in the given date range.</string>
        </property>
        <property name="text">
         <string>&amp;Events</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="journalsCheck">
        <property name="toolTip">
         <string>Search for journal entries with matching criteria.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to search for journal entries that match the search pattern and have start dates in the given date range.</string>
        </property>
        <property name="text">
         <string>&amp;Journals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="todosCheck">
        <property name="toolTip">
         <string>Search for to-dos with matching criteria.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to search for to-dos that match the search pattern and have due dates in the given date range.</string>
        </property>
        <property name="text">
         <string>To-&amp;dos</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="searchGroup">
     <property name="title">
      <string>Search In</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="categoryCheck">
        <property name="toolTip">
         <string>Include tags in your search.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to match tags against your search pattern.</string>
        </property>
        <property name="text">
         <string>Ta&amp;gs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="locationCheck">
        <property name="toolTip">
         <string>Include locations in your search.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to match locations against your search pattern.</string>
        </property>
        <property name="text">
         <string>Locations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="descriptionCheck">
        <property name="toolTip">
         <string>Include descriptions in your search.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to match descriptions against your search pattern.</string>
        </property>
        <property name="text">
         <string>Desc&amp;riptions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="summaryCheck">
        <property name="toolTip">
         <string>Include summaries in your search.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to match summaries against your search pattern.</string>
        </property>
        <property name="text">
         <string>Su&amp;mmaries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="attendeeCheck">
        <property name="toolTip">
         <string>Include attendees in your search.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to match attendees against your search pattern.</string>
        </property>
        <property name="text">
         <string>Attendees</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="dateRangeGroup">
     <property name="title">
      <string>Date Range</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="dateRangeCheckbox">
        <property name="toolTip">
         <string>When checked, the search will be limited to the specified date range.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box to search for events within a specified date range. If the box is unchecked, all events will be searched.</string>
        </property>
        <property name="text">
         <string>Limit search to specified date range</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="startDateLabel">
          <property name="text">
           <string>Fr&amp;om:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>startDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="startDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The starting date of the period to search.</string>
          </property>
          <property name="whatsThis">
           <string>Search for items in the time period starting at the date given here.</string>
          </property>
          <item>
           <property name="text">
            <string>2008-08-18</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="endDateLabel">
          <property name="text">
           <string>&amp;To:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>endDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="endDate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>The ending date of the period to search.</string>
          </property>
          <property name="whatsThis">
           <string>Search for items in the time period ending at the date given here.</string>
          </property>
          <item>
           <property name="text">
            <string>2008-08-18</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="inclusiveCheck">
        <property name="toolTip">
         <string>Display events only if their start date and end date are both in the date range specified.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you only want to find events that occur entirely within the date range specified.  Events that start during the date range but end after it, or end during the date range but start before it, will be excluded from the search results.</string>
        </property>
        <property name="text">
         <string>Exclude e&amp;vents that extend outside the specified date range</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="includeUndatedTodos">
        <property name="toolTip">
         <string>Include to-dos that have a start date, due date,  or completion date in the date range specified, or have neither a start date nor an end date.</string>
        </property>
        <property name="whatsThis">
         <string>Check this box to search for to-dos that have a start date, due date, or completion date in the date range specified, or have neither a start date nor an end date.  If the box is unchecked, only to-dos that have a due date in the date range will be displayed.</string>
        </property>
        <property name="text">
         <string>Include all to-dos &amp;within the specified date range</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="unfiltered">
        <property name="toolTip">
         <string>Uncheck this box to skip incidences matching the currently selected view filter</string>
        </property>
        <property name="whatsThis">
         <string>When this box is checked all incidences will be considered in the search, even those that aren't displayed according the currently selected view filter</string>
        </property>
        <property name="text">
         <string>Include all incidences matched by view filter</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="numItems">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="listViewFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>eventsCheck</tabstop>
  <tabstop>journalsCheck</tabstop>
  <tabstop>todosCheck</tabstop>
  <tabstop>startDate</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>inclusiveCheck</tabstop>
  <tabstop>includeUndatedTodos</tabstop>
  <tabstop>categoryCheck</tabstop>
  <tabstop>locationCheck</tabstop>
  <tabstop>descriptionCheck</tabstop>
  <tabstop>summaryCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
